test/api-test.js

```diff
@@ -224,8 +235,10 @@ describe('RippleAPI', function() {
 
   it('getOrderbook - direction is correct for bids and asks', function(done) {
     this.api.getOrderbook(address, orderbook, {}, (error, data) => {
-      assert(_.every(data.bids, bid => bid.specification.direction === 'buy'));
-      assert(_.every(data.asks, ask => ask.specification.direction === 'sell'));
+      assert.ok(
```

Comment:
Same as just assert().
Turned out that I has old assert-diff installed where just assert() doesn't work.
I will change assert.ok back to normal in next commit.